NAMESPACE

```diff
@@ -74,6 +74,7 @@ export(mcmc_nuts_treedepth)
 export(mcmc_pairs)
```

Comment:
This PR seems to contain no tests for the added functions. Please add some tests as well.

Reply:
I have now added tests for the plotting functions, ppc_pit_ecdf, ppc_pit_ecdf_grouped, and mcmc_rank_ecdf, as well as the underlying gamma_adjustment functions.